include/deal.II/numerics/vector_tools_constraints.h

```diff
@@ -295,16 +295,29 @@ namespace VectorTools
    * @see
    * @ref GlossBoundaryIndicator "Glossary entry on boundary indicators"
    */
+#ifndef _MSC_VER
   template <int dim, int spacedim>
   void
   compute_no_normal_flux_constraints(
-    const DoFHandler<dim, spacedim> &   dof_handler,
+    const DoFHandler<dim, spacedim>    &dof_handler,
     const unsigned int                  first_vector_component,
     const std::set<types::boundary_id> &boundary_ids,
-    AffineConstraints<double> &         constraints,
-    const Mapping<dim, spacedim> &      mapping =
+    AffineConstraints<double>          &constraints,
+    const Mapping<dim, spacedim>       &mapping =
       (ReferenceCells::get_hypercube<dim>()
          .template get_default_linear_mapping<dim, spacedim>()));
+#else
+  template <int dim, int spacedim>
+  void
+  compute_no_normal_flux_constraints(
+    const DoFHandler<dim, spacedim>    &dof_handler,
+    const unsigned int                  first_vector_component,
+    const std::set<types::boundary_id> &boundary_ids,
+    AffineConstraints<double>          &constraints,
+    const Mapping<dim, spacedim>       &mapping =
+      (ReferenceCells::get_hypercube<dim>()
+         .ReferenceCell::get_default_linear_mapping<dim, spacedim>()));
```

Comment:
... and this is the second change I could see.

Reply:
I close this for the formatting problem (sorry for that), and open a new one.